### PR TITLE
github: fix broken templeate

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 

None

### What is changed and how it works?

fix broken templeate
### Check List 

Tests 

 - No code

Code changes

None

Side effects

None

Related changes
None
### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
tested on my repo:

image


Last time it was because my IDE auto-formatted it, sorry!

**Comment by hi-rustin** [comment]
/uncc lonng

/cc @amyangfei 

Could you please take a look? Thanks!

File: .github/ISSUE_TEMPLATE/flaking-test.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
-name: Flaking Test about: Report flaky tests or jobs in TiCDC CI labels: component/test
+name: Flaking Test
```
Comment (asddongmen) [inline .github/ISSUE_TEMPLATE/flaking-test.md:2]: may be we can add a cute emoj here~

Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/flaking-test.md:2]: I would love to add a 😈 because it's so much and hard to fix.

Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/flaking-test.md:2]: or 😢 It can express our feelings about it.

**Comment by hi-rustin** [comment]
added
image

**Review by asddongmen — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**